src/interpreter/plugin/SimpleArithmertic.ts

```diff
@@ -0,0 +1,217 @@
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UNARY_PERCENT': {
+      method: 'unary_percent',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UPLUS': {
+      method: 'uplus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+  }
+
+  public add(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.ADD'),
+      (a: number, b: number) => this.interpreter.arithmeticHelper.addWithEpsilon(a, b)
+      )
+  }
+
+  public concat(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.CONCAT'),
+      (a: string, b: string) => a.concat(b)
+    )
+  }
+
+  public divide(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.DIVIDE'),
+      divide
+    )
+  }
+
+  public eq(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.EQ'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) === 0
+      )
+    )
+  }
+
+  public gt(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.GT'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) > 0
+      )
+    )
+  }
+
+  public gte(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.GTE'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) >= 0
+      )
+    )
+  }
+
+  public lt(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.LT'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) < 0
+      )
+    )
+  }
+
+  public lte(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.LTE'),
+      (a: InternalNoErrorCellValue, b: InternalNoErrorCellValue) => (
+        this.interpreter.arithmeticHelper.compare(a, b) <= 0
+      )
+    )
+  }
+
+  public minus(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.MINUS'),
+      (a: number, b: number) => this.interpreter.arithmeticHelper.subtract(a, b)
+    )
```

Comment:
Would passing a reference to the helper method be simple and enough?
```suggestion
    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.MINUS'), this.interpreter.arithmeticHelper.subtract)
```